packages/react-components/src/localization/locales/fr-FR/strings.json

```diff
@@ -76,6 +89,21 @@
     "unmuteGeneric": "Échec de l’activation du son du microphone",
     "speakingWhileMuted": "Votre micro est désactivé",
     "startScreenShareGeneric": "Échec du démarrage du partage d’écran",
-    "stopScreenShareGeneric": "Échec de l’arrêt du partage d’écran"
+    "stopScreenShareGeneric": "Échec de l’arrêt du partage d’écran",
+    "callNetworkQualityLow": "La qualité de réseau est faible.",
+    "callNoSpeakerFound": "Aucun haut-parleur ou casque n’a été trouvé. Connectez un périphérique audio pour entendre l’appel.",
+    "callNoMicrophoneFound": "Aucun micro n’a été trouvé. Connectez un périphérique d’entrée audio.",
+    "callMicrophoneAccessDenied": "Impossible d’accéder au microphone. Cliquez sur le verrou dans la barre d’adresses pour accorder l’autorisation à cette page Web.",
+    "callMicrophoneMutedBySystem": "Votre micro est désactivé par votre système.",
+    "callMacOsMicrophoneAccessDenied": "Impossible d’accéder au microphone. Accordez l’autorisation de microphone dans vos paramètres de confidentialité macOS.",
+    "callLocalVideoFreeze": "La bande passante réseau est médiocre. Votre vidéo peut apparaître suspendue pour les autres participants à l’appel.",
+    "callCameraAccessDenied": "Impossible d’accéder à la caméra. Cliquez sur le verrou dans la barre d’adresses pour accorder l’autorisation à cette page Web.",
+    "callCameraAlreadyInUse": "Impossible d’accéder à la caméra. Il est peut-être déjà utilisé par une autre application.",

```

Comment:
OMG, "Il est peut-être déjà utilisé ..." Caméra is female, so "**Elle** est peut-être déjà **utilisée** ... "